.github/workflows/jerigon-native.yml

```diff
@@ -58,41 +57,37 @@ jobs:
 
       #It is much easier to use cast tool in scripts so install foundry
       - name: Install Foundry
-        uses: foundry-rs/foundry-toolchain@v1          
+        uses: foundry-rs/foundry-toolchain@v1
 
       - name: Run cancun test network
         run: |
           docker pull ghcr.io/0xpolygonzero/erigon:feat-zero
-          kurtosis run --enclave cancun-testnet github.com/ethpandaops/ethereum-package@4.0.0 --args-file jerigon-test-network/network_params.yml  
+          kurtosis run --enclave cancun-testnet github.com/ethpandaops/ethereum-package@4.0.0 --args-file jerigon-test-network/network_params.yml
 
       - name: Generate blocks with transactions
         run: |
-            ETH_RPC_URL="$(kurtosis port print cancun-testnet el-2-erigon-lighthouse ws-rpc)"
-            cast rpc eth_blockNumber --rpc-url $ETH_RPC_URL
-            cd jerigon-test-network && set -a && source .env && set +a
-            bash ./tests/generate_transactions.sh
- 
+          ETH_RPC_URL="$(kurtosis port print cancun-testnet el-2-erigon-lighthouse ws-rpc)"
+          cast rpc eth_blockNumber --rpc-url $ETH_RPC_URL
+          cd jerigon-test-network && set -a && source .env && set +a
+          bash ./tests/generate_transactions.sh
+
       - name: Run prove blocks with native tracer in test_only mode
         run: |
           ETH_RPC_URL="$(kurtosis port print cancun-testnet el-2-erigon-lighthouse ws-rpc)"
           cd zero_bin/tools
           ulimit -n 8192
-          OUTPUT_TO_TERMINAL=true ./prove_rpc.sh 0x5 0xf $ETH_RPC_URL native true 3000 100 test_only
+          OUTPUT_TO_TERMINAL=true ./prove_rpc.sh 0x1 0xf $ETH_RPC_URL native true 3000 100 test_only
```

Comment:
I think we should start from block 0x2, until we decide what to do with Erigon or we skip alloy and implement manual parsing of `eth_getProof` json result. This would block other PRs

Reply:
Fixed in erigon upstream